modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/PageRenderer/index.js

```diff
@@ -180,8 +181,11 @@ const Renderer = ({
 	const variant = getVariant({page, pages, paginationMode, view, viewMode});
 	const variantComponents = LAYOUT_COMPONENTS_TYPES[variant] || {};
 
+	const formBuilderVariant = editable ? EditorVariant : {};
```

Comment:
Here We avoid using `editable` property thru DefaultVariant and use DefaultVariant for just rendering things without being responsible for Drag/Drop and other Builder business logics :)